docs/testrunner-toolkit/saucectl.md

```diff
@@ -1,188 +1,355 @@
 ---
 id: saucectl
-title: Sauce CTL
-sidebar_label: CLI Reference
+title: saucectl CLI
+sidebar_label: saucectl CLI
 ---
 
-This page provides command line references for `saucectl`, the CLI tool used with [Testrunner Toolkit](testrunner-toolkit.md).
+The `saucectl` command line interface orchestrates the relationship between your tests in your framework, and the rich parallelization, test history filtering, and analytics of Sauce Labs. This page defines each of the commands and flags available in the CLI to allow you to execute your tests with precision.
 
-> Please refer to the repository [README](https://github.com/saucelabs/saucectl) for installation, development, and contribution requirements.
+See [Installation and Setup](/testrunner-toolkit/installation) for installation options.
 
-## Commands
 
-### `configure`
+## `$ saucectl init [flags]`
 
-__Description__: Sets your [Sauce Labs authentication credentials](https://app.saucelabs.com/user-settings) and generates a `credentials.yml` file. Please refer to [the installation page](/testrunner-toolkit/installation#connecting-to-sauce-labs) for further information.
+A bootstrapping command to generate a configuration file for the framework of your choice in the format required by `saucectl`. Running the command with no parameters initiates interactive mode, prompting you for relevant property values iteratively. Alternatively, you can add supported parameters inline to specify the relevant property values.
 
-__Example__:
-```bash
-saucectl configure
+In either case, the command generates a `.sauce/config.yml` folder and file in the location from which the command is run. If you have an existing project directory for your framework, it is advised that you run this command from the project root.
+
+```bash title="Interactive Example"
+$ saucectl init
+12:13:20 INF Start Init Command
+? Select region: us-west-1
+? Select framework:  [Use arrows to move, type to filter]
+> cypress
+ playwright
+ puppeteer
+ testcafe
+ espresso
+ xcuitest
+```
+
+```bash title="Batch Example"
+$saucectl init -r us-west-1 -f cypress -b chrome
 ```
 
-### `run`
+`saucectl` supports the following configuration flags as inline specifications.
+
+
+### `-a, --accessKey`
```

Comment:
Question opened for every field:
Shouldn't we say in that title that there is a parameter ?
In `saucectl init` case all have one but `--help`, and at first glance it may be not clear that a value is needed.

Was thinking of something like that:
```suggestion
### `-a <string>, --accessKey <string>`
```